spec/models/household_member_spec.rb

```diff
@@ -5,17 +5,21 @@
 RSpec.describe HouseholdMember, type: :model do
   describe ".with_inconsistent_income" do
     it "returns HouseholdMembers with inconsistent income" do
-      described_class.create!(first_name: "alice", income_consistent: false)
-      described_class.create!(first_name: "bob", income_consistent: nil)
+      described_class.create!(first_name: "alice", income_consistent: false, app: app)
+      described_class.create!(first_name: "bob", income_consistent: nil, app: app)
 
       expect(
         described_class.with_inconsistent_income.map(&:first_name),
       ).to eq(%w[alice bob])
     end
 
     it "does not return HouseholdMembers with consistent income" do
-      described_class.create!(income_consistent: true)
+      described_class.create!(income_consistent: true, app: app)
       expect(described_class.with_inconsistent_income).to be_empty
     end
   end
+
+  def app
+    @_app ||= App.create
```

Comment:
(which should be deleted eventually, too. I am just hesitating because I feel like we will most definitely be using this class in The Future)